crypto/src/commitments/kzg.rs

```diff
@@ -179,9 +179,9 @@ impl<const N: usize, F: IsPrimeField<RepresentativeType = UnsignedInteger<N>>, P
         p: &Polynomial<FieldElement<F>>,
     ) -> Self::Commitment {
         let value = p.evaluate(x);
-        let numerator = p + Polynomial::new_monomial(-&value, 0);
-        let denominator = Polynomial::new(&[-x, FieldElement::one()]);
-        self.commit(&(numerator / denominator))
+        let mut poly_to_commit = p + Polynomial::new_monomial(-&value, 0);
+        poly_to_commit.ruffini_division_inplace(x);
```

Comment:
Shouldn't this one be `-x`?

Reply:
I think it's ok. Ruffini takes an `a` as parameter to perferm division by $X - a$